Sources/Core/Formatter/SinglelineFormatter.swift

```diff
@@ -15,6 +15,20 @@ class SinglelineFormatter: PrettyFormatter {
         return "[\(contents)]"
     }
 
+    func tupleString(elements: [(String?, String)]) -> String {
+        // if the labels of tuples are not specificated, they are like ".1" (not nil).
+        // Using "." as the first charactor of the label of tuple is prohibited.
+        let labelValuePairs: [String] = elements.map { label, value in
+            if let label = label, label.first != "." {
+                return label + ": " + value
+            } else {
+                return value
+            }
+        }
+
+        return "(\(labelValuePairs.joined(separator: ", ")))"
```

Comment:
Is this easier to read?

```suggestion
        return "(" + labelValuePairs.joined(separator: ", ") + ")"
```